opencti-platform/opencti-graphql/src/cyio/schema/assets/network/sparql-query.js

```diff
@@ -41,59 +59,111 @@ export function getReducer( type ) {
+    binding: function (iri, value) { return parameterizePredicate(iri, value ? `"${value}"`: null, this.predicate, "locations")},
+    optional: function (iri, value) { return optionalizePredicate(this.binding(iri, value))}
+  },
+  asset_tag: {
+    predicate: "<http://scap.nist.gov/ns/asset-identification#asset_tag>",
+    binding: function (iri, value) { return parameterizePredicate(iri, value ? `"${value}"`: null, this.predicate, "asset_tag")},
+    optional: function (iri, value) { return optionalizePredicate(this.binding(iri, value))}
+  },
+  serial_number: {
+    predicate: "<http://scap.nist.gov/ns/asset-identification#serial_number>",
+    binding: function (iri, value) { return parameterizePredicate(iri, value ? `"${value}"`: null, this.predicate, "serial_number")},
+    optional: function (iri, value) { return optionalizePredicate(this.binding(iri, value))}
+  },
+  vendor_name: {
+    predicate: "<http://scap.nist.gov/ns/asset-identification#vendor_name>",
+    binding: function (iri, value) { return parameterizePredicate(iri, value ? `"${value}"`: null, this.predicate, "vendor_name")},
+    optional: function (iri, value) { return optionalizePredicate(this.binding(iri, value))}
+  },
+  version: {
+    predicate: "<http://scap.nist.gov/ns/asset-identification#version>",
+    binding: function (iri, value) { return parameterizePredicate(iri, value ? `"${value}"`: null, this.predicate, "version")},
+    optional: function (iri, value) { return optionalizePredicate(this.binding(iri, value))}
+  },
+  release_date: {
+    predicate: "<http://scap.nist.gov/ns/asset-identification#release_date>",
+    binding: function (iri, value) { return parameterizePredicate(iri, value ? `"${value}"^^xsd:datetime`: null, this.predicate, "release_date")},
+    optional: function (iri, value) { return optionalizePredicate(this.binding(iri, value))}
+  },
+  network_id: {
+    predicate: "<http://scap.nist.gov/ns/asset-identification#network_id>",
+    binding: function (iri, value) { return parameterizePredicate(iri, value ? `"${value}"`: null, this.predicate, "network_id")},
+    optional: function (iri, value) { return optionalizePredicate(this.binding(iri, value))}
+  },
+  network_name: {
+    predicate: "<http://scap.nist.gov/ns/asset-identification#network_name>",
+    binding: function (iri, value) { return parameterizePredicate(iri, value ? `"${value}"`: null, this.predicate, "network_name")},
+    optional: function (iri, value) { return optionalizePredicate(this.binding(iri, value))}
+  },
+  network_address_range: {
+    predicate: "<http://scap.nist.gov/ns/asset-identification#network_address_range>",
+    binding: function (iri, value) { return parameterizePredicate(iri, value ? `"${value}"`: null, this.predicate, "network_address_range")},
+    optional: function (iri, value) { return optionalizePredicate(this.binding(iri, value))}
+  },
+}
 
-const inventoryConstraint = `
```

Comment:
with the inventoryConstraint missing, it is not limiting the query to return only those networks in the inventory.  I didn't see where there was some place else, such as in assetUtils.js were that might be contained.

Reply:
Added in latest push